packages/quantic/copy-static-resources.js

```diff
@@ -9,10 +9,14 @@ const copy = async (source, dest) => {
 const main = async () => {
   console.info('Begin copy.');
 
-  await copy(
-    './node_modules/@coveo/headless/dist',
-    './force-app/main/default/staticresources/coveoheadless'
-  );
+  try {
```

Comment:
I think doing a try catch is a good move here, but since the failing copy step actually "passed" in the workflow, I would throw something that would fail the step when an error is caught.

Also, the failing instruction is the `copy()`. Given this is unchanged, I'm not sure why this would fix the workflow. Could you explain your reasoning?

Reply:
Well I added this as more of an investigation. I don't know why the copy fails and the workflow error only says `This error originated either by throwing inside of an async function without a catch block,`, so I'm thinking maybe if I catch and log the error we will have some more information and then more to work with?

Reply:
All right that's fair. Next time I think it would be better if the PR description reflects what result you're aiming for so reviewers know what to expect.